openEHR_RM/es/openehr_terminology.xml

```diff
@@ -0,0 +1,322 @@
+<terminology name="openehr" language="es">	
+	<codeset issuer="openehr" openehr_id="compression algorithms" external_id="openehr_compression_algorithms">
+		<code value="compresión"/>
+		<code value="deflación"/>
+		<code value="gzip"/>
+		<code value="zlib"/>
+		<code value="otro"/>
+	</codeset>
+	<codeset issuer="openehr" openehr_id="integrity check algorithms" external_id="openehr_integrity_check_algorithms">
+		<code value="SHA-1"/>
+		<code value="SHA-256"/>
+	</codeset>	
+	<codeset issuer="openehr" openehr_id="normal statuses" external_id="openehr_normal_statuses">
+		<code value="HHH"/>
+		<code value="HH"/>
+		<code value="H"/>
+		<code value="N"/>
+		<code value="L"/>
+		<code value="LL"/>
+		<code value="LLL"/>
+	</codeset>
+	<group name="attestation reason">
+		<concept id="240" rubric="firmado"/>
+		<concept id="648" rubric="testigo"/>
+	</group>
+	<group name="audit change type">
+		<concept id="249" rubric="creación"/>
+		<concept id="250" rubric="enmienda"/>
+		<concept id="251" rubric="modificación"/>
+		<concept id="252" rubric="síntesis"/>
+		<concept id="523" rubric="eliminado"/>
+		<concept id="666" rubric="testimonio"/>
+		<concept id="253" rubric="desconocido"/>
+	</group>
+	<group name="composition category">
+		<concept id="431" rubric="persistente"/>
+		<concept id="435" rubric="episódico"/>
+		<concept id="433" rubric="evento"/>
+	</group>
+	<group name="MultiMedia">
```

Comment:
Theoretically, this group should be in the code sets - these codes are supposed to be language-independent. You can't change that in this PR, but we should flag it to SEC.

Reply:
Indeed, this section is commented out in the other language-files - so it should be also commented here.